src/pages/RepoPage/CoverageOnboarding/GitHubActions/GitHubActionsRepoToken.tsx

```diff
@@ -0,0 +1,106 @@
+import { useParams } from 'react-router-dom'
+
+import patchAndProject from 'assets/repoConfig/patch-and-project.svg'
+import { useRepo } from 'services/repo'
+import A from 'ui/A'
+import CopyClipboard from 'ui/CopyClipboard'
+
+const codecovActionString = `- name: Upload coverage reports to Codecov
+  uses: codecov/codecov-action@v3
+  env:
+    CODECOV_TOKEN: \${{ secrets.CODECOV_TOKEN }}
+`
+
+interface URLParams {
+  provider: string
+  owner: string
+  repo: string
+}
+
+function GitHubActionsRepoToken() {
+  const { provider, owner, repo } = useParams<URLParams>()
+  const { data } = useRepo({ provider, owner, repo })
+  const repoUploadToken = data?.repository?.uploadToken
+
+  return (
+    <div className="flex flex-col gap-6">
+      <div className="flex flex-col gap-4">
+        <div>
+          <h2 className="text-base font-semibold">
+            Step 1: add repository token as{' '}
+            <A
+              to={{ pageName: 'githubRepoSecrets' }}
+              isExternal
+              hook="GitHub-repo-secrects-link"
+            >
+              repository secret
+            </A>
+          </h2>
+          <p className="text-base">
+            Admin required to access repo settings {'>'} secrets and variable
+            {' >'} actions
```

Comment:
Can you swap the `>` to `&gt;`, that way we're using browser native symbols over this work around.